bin/tape

```diff
@@ -29,6 +31,15 @@ opts.require.forEach(function (module) {
     }
 });
 
+// Load .gitignore from cwd
+var ignoreStr = '';
+try {
+    ignoreStr = readFileSync(resolvePath(cwd, opts.ignore === '' ? '.gitignore' : opts.ignore), 'UTF-8');
+} catch (e) {}
```

Comment:
i think this should probably exit with an error if `opts.ignore` is provided and the file read doesn't work?